BREEZE.rst

```diff
@@ -46,409 +46,555 @@ Here is the short 10 minute video about Airflow Breeze
 
-You can specify for example python version to use, backend to use and environment
-for testing - you can recreate the same environments as we have in matrix builds in Travis CI.
+2. Run the ``docker system prune`` command.
 
-For example you could choose to run python 3.6 tests with mysql as backend and in docker
-environment by:
+3. Run ``docker images --all`` and ``docker ps --all`` to verify that your Docker is clean.
 
-.. code-block:: bash
+   Both commands should return an empty list of images and containers respectively.
 
-   ./breeze --python 3.6 --backend mysql --env docker
+If you run into disk space errors, consider pruning your Docker images with the ``docker system prune --all`` command. You may need to restart the Docker Engine before running this command.
 
-The choices you made are persisted in ``./.build/`` cache directory so that next time when you use the
-``breeze`` script, it will use the values that were used previously. This way you do not
-have to specify them when you run the script. You can delete the ``.build/`` directory in case you want to
-restore default settings.
+In case of disk space errors on macOS, increase the disk space available for Docker. See `Prerequsites <#prerequisites>`_ for details.
 
-The defaults when you run the environment are reasonable (python 3.6, sqlite, docker).
+Building the Images
+-------------------
 
-Mounting local sources to Breeze
---------------------------------
+You can manually trigger building the local images using the script:
 
-Important sources of airflow are mounted inside the ``airflow-testing`` container that you enter,
-which means that you can continue editing your changes in the host in your favourite IDE and have them
-visible in docker immediately and ready to test without rebuilding images. This can be disabled by specifying
-``--skip-mounting-source-volume`` flag when running breeze, in which case you will have sources
-embedded in the container - and changes to those sources will not be persistent.
+.. code-block::
 
+  ./scripts/ci/local_ci_build.sh
 
-After you run Breeze for the first time you will have an empty directory ``files`` in your source code
-that will be mapped to ``/files`` in your docker container. You can pass any files there you need
-to configure and run docker and they will not be removed between docker runs.
+The scripts that build the images are optimized to minimize the time needed to rebuild the image when
+the source code of Airflow evolves. This means that if you already have the image locally downloaded and built,
+the scripts will determine whether the rebuild is needed in the first place. Then the scripts will make sure that minimal
+number of steps are executed to rebuild parts of the image (for example, PIP dependencies) and will give
+you an image consistent with the one used during Continuous Integration.
 
-Running tests in Airflow Breeze
--------------------------------
+Pulling the Latest Images
```

Comment:
I merged sections on pulling the latest images (describes a breeze flag) with the section on force-pulling the images (describes a script). Please check whether I did that correctly

Reply:
Looks good ! 👍